### ISSUE TITLE
Publish script should disable Jekyll

### ISSUE THREAD

**Issue by WasabiFan** [ev3dev/ev3dev.github.io#162]
I believe there's a system for disabling Jekyll (something along the lines of adding a `.nojekyll` file). The publish script should do that so that GitHub doesn't try to build the static HTML.

**Comment by dlech** [comment]
Is this actually a problem? The way I understand it, it won't try to build anything that does not have "frontmatter"

**Comment by WasabiFan** [comment]
The main issue I want to bypass is that it currently takes 5-15 minutes to wait for the build to queue and finish, even though the build isn't doing anything; my hope is that disabling Jekyll will shorten that delay. Having GitHub run the build also means that files starting with underscores or some other characters are stripped, which isn't helping anything either.

The change should be a one-liner, so my intention with opening this issue was not to ask for help; I just need to remember to do it.

**Comment by dlech** [comment]
I'm wondering if this is something I should also be doing for html docs that I publish using github pages (example: ev3devKit). I haven't ever noticed any build queue or delays in publishing. So, I'm curious what you are seeing that I am not.

**Comment by WasabiFan** [comment]
Yeah, my thinking is basically this: If it isn't a Jekyll site, prevent it from being treated as one. Although right now there shouldn't be much in the way of technical restrictions that make static files break, there could be something we're unaware of or that develops in the future that catches us off-guard.

>  I haven't ever noticed any build queue or delays in publishing. So, I'm curious what you are seeing that I am not.

The first time I really was impacted by it was last night; I published my test page for #163 and waited at least 15 minutes and the site hadn't updated. It was at that point that I just posted the PR with a note that it didn't seem to have updated yet. I came back another 20 minutes later, and everything had updated, so I edited it out.

I just checked [status.github.com] and it looks like there was a service outage early today, but it doesn't say whether it affects Pages or not and the time of initial post was an hour or 3 after I saw this, so I don't think that it was the cause (although it could be).
